### ISSUE TITLE
Mail Templates: Post on Profile does not use updated [footer] template

### ISSUE THREAD

**Issue by auntialias** [SocialEngine/phpv4-issues#285]
Social Engine 4.8.9 | Updating Footer mail template does not update the footer in the **Post on Profile** email that's automaticaly generated. 

I made changes today to Mailing Templates. Not all of them applied where I expected them to apply. Control Panel > Settings > Mail Settings
## Steps I went through:
1. Updated **Footer (Members) template**. (Before: Best Regards, The XXXXXXX Social Network) to include a little disclaimer at bottom (After: added new paragraph, smaller type: _Note: do not reply…_)
2. Wrote on the profile wall of another member to test new sender email address (which works fine). Discovered that footer of message does not work.

(The updated footer DID work on the **Comment** email template. 

**I have not conducted a test of ALL email templates because I do not believe it is my job to perform SocialEngine's QA and regression testing for it, especially for a paid product!**

I **have** tested and confirmed the profile wall multiple times by posting comments to wall 3 or 4 times.
## Screenshots of my Mail Templates:

This is the template after I made the change described in step 1, above.

![google chrome-se-bugreport-001](https://cloud.githubusercontent.com/assets/1919971/10719861/e19f60be-7b4d-11e5-8fa9-6a364a11980a.png)

Here is the mail template for Profile Wall. It includes the [footer] blurb.

![google chrome-se-bugreport-002](https://cloud.githubusercontent.com/assets/1919971/10719862/e19f7d38-7b4d-11e5-8c2a-60a28ab3ef5c.png)
### Resulting email message screenshot.

Note that it DOES contain the Best regards portion, but it DOES NOT contain the added note. 

![mail-se-bugreport-001](https://cloud.githubusercontent.com/assets/1919971/10719859/e19b11b2-7b4d-11e5-9d85-7085c1b6e9f0.png)
### Footer Non Member template

(Once I discovered the problem, I updated it _just in case_ SE was using IT instead of the Members Template. Nope. No change. It's here for the sake of thoroughness and to answer the "did you try…?" question. 
![google chrome-se-bugreport-003](https://cloud.githubusercontent.com/assets/1919971/10719860/e19ba384-7b4d-11e5-8d87-7e60efe7dde5.png)

**Comment by auntialias** [comment]
Well, I found out where the footer was drawing from... a stray entry in the Language Editor.
(there are two identical entries with the basic footer message. It was the 2nd one that was being displayed. (I appended each Test A Test B to find out.)

Since I found out 2nd one (test B) was the offender, I tested to see what would happen if I replaced the entry with [footer]. Then tried to test it again. Result: email not sent. tested again, email not sent.

Only when I re-entered old copy was the email sent. Strangely enough, once the notification of test (footer = re-entered old copy), then the two stuck test emails went through.

**QUESTION: How is it possible to figure out the connection with the entry in the Language Editor with the part of SocialEngine that calls on it?**

And where oh where is it documented that once you change the footer text, you'll need to additionally edit a random snippet of language in order to make sure that **that**, too, is updated.

![google chrome-se-bugreport-011](https://cloud.githubusercontent.com/assets/1919971/10803348/fe13b550-7d7e-11e5-8a24-05ec57d4d51a.png)
